assets/js/common/SuseManagerSettingsDialog/SuseManagerSettingsModal.jsx

```diff
@@ -54,37 +54,49 @@ function SuseManagerSettingsModal({
 
   return (
     <Modal title="Enter SUSE Manager Settings" open={open} onClose={onCancel}>
-      <div className="grid grid-cols-6 items-center my-5 gap-6">
+      <div className="grid grid-cols-6 my-5 gap-6">
         <Label className="col-span-2" required>
           SUSE Manager URL
         </Label>
-        <Input
-          className="col-span-4"
-          value={url}
-          placeholder="Enter a URL"
-          error={hasError('url', errors)}
-          onChange={({ target: { value } }) => {
-            setUrl(value);
-            onClearErrors();
-          }}
-        />
+        <div className="col-span-4">
```

Comment:
What about embedding the error in the input component?

```js
<Input
  value={url}
  placeholder="Enter a URL"
  error={getError('url', errors)}
  onChange={({ target: { value } }) => {
    setUrl(value);
    onClearErrors();
  }}
/>
```
Really up to you :smile: There are just many `hasError` and related `getError`.
I am fine either way